source/_integrations/apple_tv.markdown

```diff
@@ -10,165 +10,229 @@ ha_iot_class: Local Push
 ha_release: 0.49
 ---
 
-The `apple_tv` platform allows you to control an Apple TV (3rd and 4th generation). See the [remote platform](/integrations/apple_tv#remote) if you want to send remote control buttons, e.g., arrow keys.
+The `apple_tv` integration allows you to control an Apple TV (any generation). See the
+[remote platform](/integrations/apple_tv#remote) if you want to send remote control buttons,
+e.g., arrow keys.
 
 There is currently support for the following device types within Home Assistant:
 
 - Media Player
 - [Remote](#remote)
 
-<div class='note'>
-Currently, you must have Home Sharing enabled for this to work. Support for pairing Home Assistant with your device will be supported in a later release.
-</div>
-
 ## Configuration
 
-To use this component, you must first install some system libraries and a compiler. For Debian or a similar system, this should be enough:
+You can setup a new Apple TV either the old fashioned way in `configuration.yaml` or just
+follow a simple configuration flow ("Normal Setup"). Instructions for both methods are below.
+If you are unsure about which method to use, go for "Normal Setup".
+
+### Depdendencies
```

Comment:
Just remove this section. Users running a venv should be capable of knowing how to handle this. Our documentation should focus on Home Assistant and Home Assistant Core on Docker installations.

(which makes this section incorrect as well, since this doesn't apply to Home Assistant Core on Docker either).